### ISSUE TITLE
image format definition based on formats-1.0.xsd in docs

### ISSUE THREAD

**Issue by oafx** [sulu/sulu-docs#311]
Hi,

the docs section [Book - Image Formats](http://docs.sulu.io/en/latest/book/image-formats.html) is still using the formats-1.0.xsd schema definition as example while i suppose it should be an example for the formats-1.1.xsd?

--

btw. if the schemaLocation is specified as
```xml
<formats xmlns="http://schemas.sulu.io/media/formats"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://schemas.sulu.io/media/formats http://schemas.sulu.io/media/formats-1.1.xsd">
``` 
shouldn't there be a file at [http://schemas.sulu.io/media/formats-1.1.xsd](http://schemas.sulu.io/media/formats-1.1.xsd) ?

regards,
chris

**Comment by danrot** [comment]
You are right, we are about to upload that file. In the mean time you find it at: https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/MediaBundle/Media/FormatLoader/schema/formats/formats-1.1.xsd

Please feel free to create a PR which adapts this file in the documentation :-)

**Comment by alexander-schranz** [comment]
fixed in #375.
xsd link also work again.
